XLS-0033d-multi-purpose-tokens/README.md

```diff
@@ -140,11 +140,15 @@ An asset scale is the difference, in orders of magnitude, between a standard uni
 
 ###### 2.1.1.2.5. `MaximumAmount`
 
-This value is an unsigned number that specifies the maximum number of MPTs that can be distributed to non-issuing accounts (i.e., `minted`). For issuances that do not have a maximum limit, this value should be set to 0xFFFFFFFFFFFFFFFF.
+An unsigned 64-bit number that specifies the maximum number of MPTs that can be distributed to non-issuing
+accounts (i.e., `minted`). The maximum value of this field is `0xFFFF'FFFF'FFFF'FFFF`.
```

Comment:
```suggestion
accounts (i.e., `minted`). The default and maximum value is 0x7FFF'FFFF'FFFF'FFFF. However, this may be subject to change in the future, so it's recommended to specify a custom value if a specific limit is needed.
```

something along these lines